pylint/checkers/refactoring.py

```diff
@@ -121,6 +109,28 @@ def get_curline_index_start():
     return False
 
 
+def _is_call_of_name(node: astroid.node_classes.NodeNG, name: str) -> bool:
```

Comment:
@PCManticore: Do we still support python 3.4? (Type hinting would not be available.).

Reply:
Answering my own question, we don't : https://github.com/PyCQA/pylint/blob/master/pylint/__pkginfo__.py#L67